src/universal/components/CreateCard/CreateCard.js

```diff
@@ -80,18 +69,41 @@ CreateCard.propTypes = {
   styles: PropTypes.object
 };
 
-const labelStyles = {
-  display: 'inline-block',
+const borderRadius = '.1875rem';
```

Comment:
A lot of our defaults are based on `1rem` which is `16px` —using certain sizes can keep things consistent (and are supposed to scale up/down via zoom nicely)...